### ISSUE TITLE
I can't stop the video once is started or see any option of the videoplayer menu.

### ISSUE THREAD

**Issue by carlosricardocm** [asciidisco/plugin.video.netflix#487]
*I'm submitting a ...*
  - [] bug report
  - [ ] feature request
  - [X] support request

## General information

I'm running kodi 18 in raspbian strech in a raspberry pi b 3

### Prerequisites

* [X] Are you running the latest version?
0.13.13

### Description

[Description of the bug or feature]

I cannot stop the videoplayer once the serie/movie is started o even call someone of the video player options for stop, play, subtitles, etc.

### Steps to Reproduce

1. [First Step]
Install the plugin version 0.13.13
2. [Second Step]
Play any serie/movie

**Expected behavior:** [What you expected to happen]
Stop the video or showing the videoplayer menu.

**Actual behavior:** [What actually happened]
Fade out screen to black

## Context (Environment)
I was able to compile and install kodi 18.0.BETA3 in raspbian stretch and install it some binary addons (inputstream.adaptive and inputstream.rtmp) in order to run this pluging.

### Installation

* [ ] I installed the plugin via zip from the Releases page
* [X] I´m using the Netflix Repo
* [ ] I´m using a different source (Please tell which)

### Operating System

* [ ] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [X] Linux (ARM)
* [ ] Android

#### Additional informatin on the environment

I installed Retropie 4.4 which is based on Raspbian Stretch and compile Kodi 18 with the binary extra addons instead of the Kodi 17.3 available in this distribution.

**Comment by xilopaint** [comment]
I have the same issue. Have you found a fix?

**Comment by carlosricardocm** [comment]
Yes, the problem was the kodi default skin if you change it to aeon nox for example the menu it's fixed.

**Comment by xilopaint** [comment]
> Yes, the problem was the kodi default skin if you change it to aeon nox for example the menu it's fixed.

It worked! How can this issue has not been reported yet? It's weird, there's no indication in README about any incompatibility between the add-on and the default skin.

I also have Kodi installed from RetroPie. Could the issue has anything to do with it?

**Comment by carlosricardocm** [comment]
I don't know, maybe this should be posted in the kodi forum insted here because  the problem isn't the plugin itself but the kodi skin, I don't have much time to find a solution but I think the issue is related with the size of the pictures, because only happens in some screen resolutions, for example in my small monitor the plugin works fine but in my 55' smart tv won't)

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by xilopaint** [comment]
Do you have an idea of what resolution can work with the default skin?

**Comment by carlosricardocm** [comment]
Sorry, I did the test a long time ago.
But, you can test it in smaller resolutions.
